csharp/test/Drivers/Snowflake/SnowflakeTestConfiguration.cs

```diff
@@ -71,5 +71,17 @@ internal class SnowflakeTestConfiguration : TestConfiguration
         /// </summary>
         [JsonPropertyName("authenticationTokenPath"), JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
         public string AuthenticationTokenPath { get; set; }
+
+        /// <summary>
+        /// The snowflake copy command. E.g. "copy into {0} from @%{1} file_format = (type = 'parquet');"
+        /// </summary>
+        [JsonPropertyName("copyCommand")]
+        public string copyCommand { get; set; }
+
+        /// <summary>
+        /// The snowflake stage command. E.g. "create or replace stage {0} url = {1} credentials = (AZURE_SAS_TOKEN = {2}) file_format = (type = 'parquet');"
+        /// </summary>
+        [JsonPropertyName("stageCommand")]
+        public string stageCommand { get; set; }
```

Comment:
The property should be named `StageCommand` to match the other naming conventions.